packages/app/src/app/pages/common/Modals/SubscriptionCancellation/SubscriptionCancellation.tsx

```diff
@@ -39,16 +39,20 @@ export const SubscriptionCancellationModal: React.FC = () => {
           <Stack justify="flex-end">
             <IconButton name="cross" title="Close" onClick={handleCloseModal} />
           </Stack>
-          <Text
-            as="h2"
-            size={32}
-            color="#FFFFFF"
-            align="center"
-            css={{ marginTop: 0 }}
-            fontFamily="everett"
-          >
-            You&apos;ll lose access to all Pro features
-          </Text>
+          <Stack justify="center">
```

Comment:
It didn't :(

We also wanted to add the `400px` max width. But thinking about it now, it might have helped if we added `text-align` `center` in the parent.